programs/add_numbers.py

```diff
@@ -0,0 +1,4 @@
+x = int(input("enter the x number : "))
+y = int(input("enter the y number :"))
+sum = x+y
+print("sum_of_numbers : " + str(sum) )
```

Comment:
remove underscore in `sum_of_numbers`

Reply:
addvalues_underscore removed,kindly check